Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
+
+COPY . /app/phabricator/src/extensions/
```

Comment:
version.json is *a* way to handle this, but since the code isn't going to actually be "served", even a text file containing the current/correct sha would work.